src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/TextBox/TextBox_Disabled.xaml

```diff
@@ -11,12 +11,12 @@
 
 	<StackPanel Margin="50">
 		<TextBox PlaceholderText="disables on letter f..."
-					 x:Name="DisableOnf"
+					 x:Name="DisableOnfTextBox"
```

Comment:
Align properties with PlaceholderText (Ctrl + K & Ctrl + D, in order to format the document) ;)